buildtest/buildsystem/singlesource.py

```diff
@@ -10,7 +10,7 @@
 
 from buildtest.config import config_opts
 from buildtest.defaults import logID
-from buildtest.utils.command import BuildTestError
+from buildtest.utils.exceptions import BuildTestError
 
```

Comment:
Can we instead do buildtest.exceptions? That is where I'd expect it.

Reply:
Sure, let me fix it.